### DISCUSSION TITLE
Best way to add system packages to user (jupyterlab) image?

### DISCUSSION THREAD

**Discussion by alimanfoo** [nebari-dev/nebari#712] [Q&A] [answered]
I'm deploying qhub for my team and would like the ability to customise the user (jupyterlab) image. This includes adding some system packages via apt. This could also include adding some jupyterlab extensions. Is there any way to do this currently (qhub 0.3.11)? 

I'm guessing I could create a new docker image based off `quansight/qhub-jupyterlab:v0.3.11`?

**Comment by viniciusdc** [comment, marked as answer]
Hi @alimanfoo thanks for oppening this discussion. QHub uses it's own registered docker images for `jupyterhub`, `dask` and `jupyterlab` services as [default](https://docs.qhub.dev/en/stable/source/02_get_started/04_configuration.html?highlight=image#default-images). You can change that behaviour building your own docker image and including the dockerhub register hash into qhub-config.
